core/server/data/migrations/versions/4.5/03-remove-unique-constraint-from-product-name.js

```diff
@@ -0,0 +1,14 @@
+const {createNonTransactionalMigration} = require('../../utils');
+
+module.exports = createNonTransactionalMigration(
+    async function up(knex) {
+        return knex.schema.table('products', (table) => {
```

Comment:
@allouis Think this should be `schema.alterTable` here instead of `schema.table`

Reply:
They're the same thing! https://github.com/knex/knex/blob/master/lib/schema/builder.js#L77

Reply:
TIL 👍